person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(id, age:, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id, :name, :age
```

Comment:
- [ ] `:name` and `:age` instance variables are the only ones that should be able to be modified and should be defined as `attr_accessors`. Only the id should be defined with `attr_reader`